kerlescan/inventory_service_interface.py

```diff
@@ -141,6 +142,15 @@ def fetch_systems_with_profiles(system_ids, service_auth_key, logger, counters):
 
     ensure_correct_system_count(system_ids, systems_result)
 
-    return interleave_systems_and_profiles(
+    systems_with_profiles = interleave_systems_and_profiles(
         systems_result, system_profiles_result, system_tags_result
     )
+
+    # remove CentOS Linux systems
+    systems_with_profiles = [
+        system
+        for system in systems_with_profiles
+        if system["operating_system"]["name"] not in FILTERED_OUT_OPERATING_SYSTEMS
```

Comment:
Is `name` always present in the `system["operating_system"]`?

Reply:
no, you are right, also `operating_system` is not always present.....

solved...